### DISCUSSION TITLE
Stay in one language - at lot of DEnglisch expressions in API

### DISCUSSION THREAD

**Discussion by TheSmartGerman** [Akkudoktor-EOS/EOS#191] [Ideas]
Hallo,
kleiner Hinweis, Bitte als positives Feedback und nicht als Kritik an der Arbeit verstehen.

Wenn hier größten teils in Englisch dokumentiert wird. Und Variablen / Übergabe Parameter auch schon in Englisch sind. Sollte man da konsequenter drauf achten. Zu Beginn ist es einfacher noch Änderungen dieser Art umzusetzen.

**Comment by drbacke** [comment]
We already have an issue due to that. We try to translate everything into english. But it takes some time

**Reply by sandboxcode** [reply]
> We already have an issue due to that. We try to translate everything into english. But it takes some time

See Issue https://github.com/Akkudoktor-EOS/EOS/issues/3 @TheSmartGerman ;)

**Comment by mattmeye** [comment]
It might also be helpful to create a glossary for technical terms. On the one hand, it helps understanding, on the other hand, it also helps to avoid synonyms.
